pkgs/games/tiltfive/control-panel.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
+, autoPatchelfHook
+, fetchurl
+, nixosTests
+
+, gtk3
+, glfw
+}:
+
+let srcs = import ./srcs.nix { inherit fetchurl; }; in
```

Comment:
Write the sources directly.

Reply:
Both `control-panel.nix`/`tiltfive-control-panel` and `driver.nix`/`tiltfive-driver` use the same source (`tiltfive_driver_${version}_amd64.unpacked_debs.tar.xz`). Inlining srcs.nix would mean having the same source (alongside version, hash, etc) declared twice, once in each file for each derivation. Plus the version would also be duplicated across three derivation sources.

For DRY, I would think it's better to keep it as is (or deduplicate it some other way), no?

Reply:
We can re-use the src and version attr of one package in another without this extra trick. We just need to find one main package and then we could do ``inherit (tiltfive-control-panel) version src;``